frontend/src/slices/auth/auth.slice.ts

```diff
@@ -22,23 +22,38 @@ const { reducer, actions, name } = createSlice({
   name: 'auth',
   reducers: {},
   extraReducers(builder) {
-    builder.addCase(signIn.fulfilled, (state, action) => {
-      state.user = action.payload;
-    });
-    builder.addCase(signUp.fulfilled, (state, action) => {
-      state.user = action.payload;
-    });
     builder.addCase(logout.fulfilled, (state) => {
       state.currentUserDataStatus = DataStatus.IDLE;
       state.user = null;
     });
-    builder.addCase(getCurrentUser.fulfilled, (state, action) => {
-      state.user = action.payload;
-      state.currentUserDataStatus = DataStatus.FULFILLED;
-    });
-    builder.addCase(usersActions.updateUser.fulfilled, (state, action) => {
-      state.user = action.payload;
+    builder.addCase(getCurrentUser.pending, (state) => {
+      state.currentUserDataStatus = DataStatus.PENDING;
     });
+    builder.addMatcher(
+      isAnyOf(
```

Comment:
```suggestion
```
avoid this for now. we will have a different approach with data statuses. write all cases explicitly